docs/ProgrammingCryptol/highAssurance/HighAssurance.tex

```diff
@@ -174,6 +174,51 @@ \subsection{Capturing test vectors}
   sure the tweak to the above example (removal of dummy single formal
   parameter) works.}
 
+%~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+\subsection{Generating test vectors}
+\label{sec:gentestvec}
+
+Cryptol has the ability to generate test vectors for simple function types using the \texttt{:dumptests} REPL command.
+When \texttt{:dumptests} is given a file to dump the output and a function in scope, it will randomly generate inputs for the function and run them for you.
+The output will contain a tab delimited file in which the first column contains the output, and each column after contains the inputs to the function.
+The command \texttt{:dumptests} uses the same settings as \texttt{:check} \ref{sec:quickcheck}, this means you can set the number of tests you want using the \texttt{:set tests=100} command.
+
+The following is an example using \href{https://github.com/GaloisInc/cryptol-specs/blob/7468d170f5e9e3913cd12e29c76d8c041ef99dd0/Primitive/Keyless/Hash/SHA2/Instantiations/SHA256.cry}{SHA2-256 implemntation} in \href{https://github.com/GaloisInc/cryptol-specs}{cryptol-specs}.
+The output file was modified slightly by putting the input on the next line tabbed in. 
```

Comment:
This sentence isn't very clear. I have two suggestions: (1) change the name of output.txt to something else (e.g. results.txt) because otherwise "outputs" and "inputs" are unconnected things and that's confusing; and (2) say why things have been changed, something like "the example results shown have been adjusted slightly for clarity by splitting each input value onto its own indented line".

Reply:
This line was no longer needed in the new example.